### DISCUSSION TITLE
Can't install WebCalendar

### DISCUSSION THREAD

**Discussion by s22-tech** [craigk5n/webcalendar#268] [Q&A]
I must be doing something wrong since no one else has brought this up, but I can't get past the install script (see [this issue for details](https://github.com/craigk5n/webcalendar/issues/265)).

I'd like to check out the rest of the script, so I need to know how to bypass the installation script for now.  The db appears to have been created properly.  Is there a setting I can change in the db or something that will let me navigate a calendar?

Removing the /install directory doesn't help - it just complains it can't find it - so hopefully someone here can help.

Thanks,
Marc

**Comment by KarlosFanta** [comment]
Which versions of PHP and Apache are you running? Tried another browser? Firefox/Chrome? Try the new version v1.9.1
Run phpinfo() to determine versions

**Reply by s22-tech** [reply]
I've tried 1.9.1, as well as the dev version mentioned in the install docs.  I've tried both Firefox and Brave.

PHP 8.0
Apache 2.4

**Reply by craigk5n** [reply]
Try the latest code from the main branch.  I put in some PHP8 fixes yesterday (so not included in 1.9.1).

If you cannot get the installer to work, you can try just using the tables-*.sql file in install/sql to create the database manually and skip the installer.  You'd have to hand-create the includes/settings.php file (the installer normally does that for you).

**Reply by s22-tech** [reply]
Finally, I'm in.  Thank you for fixing this!
